pkgs/by-name/di/distrho-ports/package.nix

```diff
@@ -16,12 +16,18 @@
 
   # empty means build all available plugins
   plugins ? [ ],
+
+  buildVST3 ? true,
+  buildLV2 ? true,
+  buildVST2 ? true,
 }:
 
 let
   rpathLibs = [
     fftwFloat
   ];
+
+  mesonPlugins = lib.mesonOption "plugins" "[${lib.concatMapStringsSep "," (x: "\"${x}\"") plugins}]";
```

Comment:
nit(not blocking): I would prefer declarations scoped closer to usages. Seems to only be used once too, not sure why it was moved up here.